styx-common/src/main/java/com/spotify/styx/storage/AggregateStorage.java

```diff
@@ -91,6 +91,12 @@ public StyxConfig config() throws IOException {
     return datastoreStorage.activeStates(componentId);
   }
 
+  @Override
+  public Optional<PersistentWorkflowInstanceState> readActiveWorkflowInstance(WorkflowInstance workflowInstance)
```

Comment:
Seems we should rename `WorkflowInstance` to `WorkflowInstanceId`. (Not for this PR.) It's confusing to see methods named `readFooBar` literally take `FooBar` as their argument.